src/com/sfuentes/AverageTest.java

```diff
@@ -0,0 +1,37 @@
+package com.sfuentes;
```

Comment:
We need to reformat a bit the method calculateAverageStudent so it will be easier to test because right now whenever you run the tests it needs an user to enter the grades of the student but all the tests must be able to run without any interaction.
I'll work with you in this one.